datafusion/sqllogictest/test_files/clickbench.slt

```diff
@@ -274,5 +274,23 @@ query PI
 SELECT DATE_TRUNC('minute', to_timestamp_seconds("EventTime")) AS M, COUNT(*) AS PageViews FROM hits WHERE "CounterID" = 62 AND "EventDate"::INT::DATE >= '2013-07-14' AND "EventDate"::INT::DATE <= '2013-07-15' AND "IsRefresh" = 0 AND "DontCountHits" = 0 GROUP BY DATE_TRUNC('minute', to_timestamp_seconds("EventTime")) ORDER BY DATE_TRUNC('minute', M) LIMIT 10 OFFSET 1000;
 ----
 
+# Clickbench "Extended" queries that test count distinct
+
+query III
+SELECT COUNT(DISTINCT "SearchPhrase"), COUNT(DISTINCT "MobilePhone"), COUNT(DISTINCT "MobilePhoneModel") FROM hits;
+----
+1 1 1
+
+query III
+SELECT COUNT(DISTINCT "HitColor"), COUNT(DISTINCT "BrowserCountry"), COUNT(DISTINCT "BrowserLanguage")  FROM hits;
+----
+1 1 1
+
+query TIIII
```

Comment:
just to verify is it okay, to benchmark a data that groups everything into a single bucket?

Reply:
I think the idea of this test is to ensure that the queries run without errors, rather than verify the results

https://github.com/apache/datafusion/blob/6e2ff2955d96cacba905d24993353c7e5fe0cf93/datafusion/sqllogictest/test_files/clickbench.slt#L19-L26

The actual input is a 13GB parquet file so I don't think it is feasible to verify the results as part of CI.